.github/workflows/test-job.yaml

```diff
@@ -85,6 +73,20 @@ jobs:
         working-directory: ./cli
         args: --timeout=3m
         skip-go-installation: true
+        skip-pkg-cache: true
+
+    - name: Install goimports
```

Comment:
any reason to move imports after linter?  @wilsonwang371

Reply:
This fixed golang lint error message we saw in previous runs

Reply:
Got it